JSONPlaceholderTests/Tests.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using JSONPlaceholderAPIClient;
+using NUnit.Framework;
+
+namespace JSONPlaceholderTests
+{
+    [TestFixture]
+    public class Tests
```

Comment:
We usually split tests into several files: GetPostsTests, PostNewPostTests, UpdatePostTests, DeletePostTests,